src/config/layers.js

```diff
@@ -665,6 +665,147 @@ export const behigParent = new BehigLayer({
   toggleLayers: [behigOk, behigNotYetOk, behigNotOk],
 });
 
+export const netzkarteEisenbahninfrastruktur = new TrafimageMapboxLayer({
+  name: 'ch.sbb.infrastruktur',
+  // copyright: '© OpenStreetMap contributors, OpenMapTiles, imagico, SBB/CFF/FFS', // TODO
+  // isBaseLayer: true, // TODO Nachfragen
+  visible: true,
+  isQueryable: false,
+  preserveDrawingBuffer: true,
+  zIndex: -1,
+  style: 'netzkarte_eisenbahninfrastruktur',
+  properties: {
+    hasInfos: true,
+    layerInfoComponent: 'InfrastrukturTopicInfo', // TODO Nachfragen
+  },
+});
+
+export const grenzen = new Layer({
+  name: 'ch.sbb.infrastruktur.grenzen.group', // TODO Alle Uebersetzungen...
+  visible: false,
+  properties: {
+    hasInfos: true,
+    description: 'ch.sbb.infrastruktur.grenzen.group-desc', // TODO Alle Uebersetzungen...
+  },
+  children: [
+    new Layer({
+      name: 'ch.sbb.infrastruktur.gemeindegrenzen.group',
+      visible: false,
+      properties: {
+        hasInfos: true,
+        description: 'ch.sbb.infrastruktur.gemeindegrenzen.group-desc',
+      },
+      children: [
+        new TrafimageGeoServerWMSLayer({
```

Comment:
Please replace by MapboxStyleLayer similar to passagierfrequenzen in order to use vector tiles

gemeinden_ch_...
kantone_ch_...

with styleLayer for grey and green borders